stable/wordpress/templates/ingress.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
     release: "{{ $.Release.Name }}"
     heritage: "{{ $.Release.Service }}"
+    {{- if .labels }}
```

Comment:
It should be: 

```
    {{- if .Values.ingress.labels }}
+{{ toYaml Values.ingress.labels | indent 4 }}
    {{- end }}
```

Reply:
Yes this is more a standard solution, but in this case we have a loop (**{{- range .Values.ingress.hosts }}**) to create a dedicated ingress per host. We have the same pattern for the annotations.